src/sentry/static/sentry/app/views/settings/project/projectKeys/projectKeyCredentials.jsx

```diff
@@ -17,6 +17,7 @@ class ProjectKeyCredentials extends React.Component {
     showDsnPublic: PropTypes.bool,
     showSecurityEndpoint: PropTypes.bool,
     showMinidump: PropTypes.bool,
+    showUnreal: PropTypes.bool,
```

Comment:
Can we reuse `showMinidump` instead of introducing a new flag? Also, where is this being populated?

Reply:
Why would unreal be different?